server/superdesk/tests.py

```diff
@@ -87,6 +87,8 @@ def drop_mongo(app):
             app.data.mongo.pymongo(prefix='LEGAL_ARCHIVE').cx.drop_database(app.config['LEGAL_ARCHIVE_DBNAME'])
         except AttributeError:
             pass
+        except pymongo.errors.ConnectionFailure:
+            raise ValueError('Invalid mongo config: uri=%s db=%s' % (app.config['MONGO_URI'], app.config['MONGO_DBNAME']))
```

Comment:
Could this error message be somewhat misleading, if for instance `ConnectionFailure` occurs because of the e.g. Mongo not running for some reason?